packages/tools/pie-icons-webc/build.js

```diff
@@ -58,6 +61,8 @@ export class ${name} extends LitElement implements IconProps {
         super.connectedCallback();
         if (this._svg?.getAttribute('width') === null) {
             const svgSize = getSvgProps('${svgClasses}', '', null, '${name}');
+            this.style.width = \`\${\`var(--icon-size-override,\${svgSize.width}\`}px)\`;
+            this.style.height = \`\${\`var(--icon-size-override,\${svgSize.height}\`}px)\`;
             this._svg?.setAttribute('width', svgSize.width);
             this._svg?.setAttribute('height', svgSize.height);
```

Comment:
Feels weird that we set here both width and height attributes and at the same time set width and height through the inline styles 🤔